lisa/operating_system.py

```diff
@@ -1610,6 +1610,17 @@ def add_repository(
 
     def _initialize_package_installation(self) -> None:
         self.wait_running_process("zypper")
+        from lisa.tools import Service
+
+
+        service = self._node.tools[Service]
+        if service.check_service_exists("guestregister"):
```

Comment:
when I use like this way, it thows 'failed. NameError: name 'Service' is not defined', @squirrelsc any idea?
```
if TYPE_CHECKING:
    from lisa.node import Node
```

Reply:
if a tool need to be referenced in os, it should be moved to base_tools. But if the tool is complex, it references other tools, they will be moved to base_tools too. For "service", there is no other dependent tools, so it should be simple.

Reply:
thanks. update